options/ansi/include/wchar.h

```diff
@@ -112,12 +113,17 @@ size_t wcsnrtombs(char *__restrict __mbs, const wchar_t **__restrict __wcs, size
 		mbstate_t *__restrict __stp);
 
 /* POSIX extensions */
+#if __MLIBC_XOPEN
 int wcwidth(wchar_t __wc);
 int wcswidth(const wchar_t *__s, size_t __size);
+#endif
+
+#if __MLIBC_POSIX2008
 wchar_t *wcsdup(const wchar_t *__s);
 int wcsncasecmp(const wchar_t *__a, const wchar_t *__b, size_t __size);
 int wcscasecmp(const wchar_t *__a, const wchar_t *__b);
 size_t wcsnlen(const wchar_t *__s, size_t __maxlen);
+#endif /* __MLIBC_POSIX2008 */
 
```

Comment:
If all these functions are POSIX extensions (as also underlined by the tests being moved there), then their implementation should be in the POSIX option IMO